auth/src/main/res/values/strings.xml

```diff
@@ -1,68 +1,71 @@
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
     <string name="app_name">ui_flow</string>
 
+    <!-- Common -->
     <string name="default_toolbar_title">@string/app_name</string>
-    <string name="title_welcome_back_password_prompt">@string/sign_in_default</string>
-    <string name="title_welcome_back_idp_prompt">@string/sign_in_default</string>
-    <string name="title_check_email">@string/sign_in_default</string>
-    <string name="title_register_email_activity">@string/create_account_title</string>
-    <string name="title_recover_password_activity">@string/recover_password_title</string>
-    <string name="title_confirm_recover_password_activity">@string/check_your_email</string>
```

Comment:
While these `title_` strings are not part of the public API, I know there are some devs that do override them.  We seem to have changed `title_confirm_recover_password_activity` to `title_confirm_recover_password` here, can we undo that just to keep the resource name as long as we can?  Or is there a reason for the change?